pymodbus/transport/transport.py

```diff
@@ -211,19 +214,25 @@ def init_setup_connect_listen(self, host: str, port: int) -> None:
         """Handle connect/listen handler."""
         if self.comm_params.comm_type == CommType.UDP:
             if self.is_server:
-                self.call_create = lambda: self.loop.create_datagram_endpoint(
+                self.call_create = lambda: cast(
```

Comment:
That is wrong, call_create is NOT loop.

Reply:
No, `self.loop` has to be a loop to call `create_datagram_endpoint`. See comment 10.